kodilla-basic-tests/src/main/java/com/kodilla/inheritance/homework/SystemWindowsOn.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.inheritance.homework;
+
+public class SystemWindowsOn extends OperatingSystem {
+    public void Chrome() {
```

Comment:
to ma byc metoda? jesli tak, to jej nazwa z malej litery. Tylko po co ta metoda?